### DISCUSSION TITLE
Colour a Ligand by B-Factor values

### DISCUSSION THREAD

**Discussion by kalraUtkarsh** [molstar/molstar#1336] [Q&A]
I have a Ligand in pdb format, I would like to paint the atoms by their respective b-factor values which are present in the 4th last  column of each line in the pdb of that Ligand. I cant seem to find a way to do so, please help

example:

HETATM 1992  C12 747 @ 103       1.638  -7.419  18.392  1.80 **0.2** LIGAND    C 0
HETATM 1993  N13 747 @ 103       0.298  -8.006  18.206  1.60  **1.4** LIGAND    N 0
HETATM 1994  O14 747 @ 103      -0.377  -4.483  17.823  1.40 **5.26** LIGAND    O 0
HETATM 1995  N15 747 @ 103       1.495  -5.921  18.252  1.60  **0.7** LIGAND    N 0

as you can see here.

**Comment by dsehnal** [comment]
Use `Atom Property -> Uncertainty/Disorder` color theme.

![image](https://github.com/user-attachments/assets/89a62d88-6c1a-4620-8fad-5ea40cb41122)
